resources/assets/components/CampaignsTable.js

```diff
@@ -27,9 +26,6 @@ const CAMPAIGNS_QUERY = gql`
           internalTitle
           pendingCount
           acceptedCount
-          causes {
-            name
-          }
```

Comment:
I removed this from the query since these values were only used for client-side filtering.